starknet_py/tests/e2e/tests_on_networks/trace_api_test.py

```diff
@@ -22,9 +22,9 @@
 @pytest.mark.asyncio
 async def test_trace_transaction(client_goerli_testnet):
     tx_to_trace: dict[type[Transaction], type[TransactionTrace]] = {
-        InvokeTransaction: InvokeTransactionTrace,
-        DeclareTransaction: DeclareTransactionTrace,
-        DeployAccountTransaction: DeployAccountTransactionTrace,
+        InvokeTransactionV1: InvokeTransactionTrace,
```

Comment:
It is also necessary to add here:
```
InvokeTransactionV0: InvokeTransactionTrace
```
There are both V0 and V1 invoke transactions in this block.

Reply:
Done